plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -256,11 +259,18 @@ export async function createRouter(
     if (scheduler && databaseTaskStore.listStaleTasks) {
       await scheduler.scheduleTask({
         id: 'close_stale_tasks',
-        frequency: { cron: '*/5 * * * *' }, // every 5 minutes, also supports Duration
+        frequency: {
+          cron:
+            options.config.getOptionalString('scaffolder.staleTasks.cron') ??
+            '*/5 * * * *',
+        }, // every 5 minutes, also supports Duration
         timeout: { minutes: 15 },
         fn: async () => {
           const { tasks } = await databaseTaskStore.listStaleTasks({
-            timeoutS: 86400,
+            timeoutS:
+              options.config.getOptionalNumber(
+                'scaffolder.staleTasks.timeout',
+              ) ?? 86400,
```

Comment:
Any new added config option needs to be represented and documented in the config.d.ts file of the package, in the same pull request.

As you do so, I recommend using our standard `HumanDuration` instead of having an implicit unit.

Like this https://github.com/backstage/backstage/blob/e0b5cd9fdba98c9559bd0e98a68276b527dbb1ae/plugins/catalog-backend/config.d.ts#L171

And consumed like this using the helper function https://github.com/backstage/backstage/blob/e0b5cd9fdba98c9559bd0e98a68276b527dbb1ae/plugins/catalog-backend/src/service/CatalogBuilder.ts#L789

I'd say in the end, maybe `scaffolder.taskTimeout` is a better name for the key too.

So users would end up writing for example

```yaml
scaffolder:
  taskTimeout: { minutes: 10 }
```

After you have read out the humanDuration at the top of this function, you can use luxon here to make seconds out of it. 

```ts
import { Duration } from 'luxon';

/* ... */
timeoutS: Duration.fromObject(humanDuration).as('seconds')
```